pkg/scheduler/apis/config/v1beta1/defaults_test.go

```diff
@@ -224,6 +230,7 @@ func TestSchedulerDefaults(t *testing.T) {
 				HealthzBindAddress: pointer.StringPtr(":12345"),
 			},
 			expected: &v1beta1.KubeSchedulerConfiguration{
+				Parallelism:        16,
```

Comment:
add a test that sets a non-default parallelism?

Reply:
+1